lib/mp4/transmuxer.js

```diff
@@ -709,14 +708,21 @@ CoalesceStream = function(options, metadataStream) {
     // important information required for the construction of
     // the final segment
     this.pendingTracks.push(output.track);
-    this.pendingBoxes.push(output.boxes);
     this.pendingBytes += output.boxes.byteLength;
 
+    // TODO: is there an issue for this against chrome?
```

Comment:
This is what caused the test change, and a lot of the headache I was seeing getting this to work